src/main/java/com/zaxxer/hikari/pool/PoolBase.java

```diff
@@ -125,14 +126,14 @@ boolean isConnectionAlive(final Connection connection)
    {
       try {
          if (isUseJdbc4Validation) {
-            return connection.isValid((int) MILLISECONDS.toSeconds(Math.max(1000L, validationTimeout)));
+            return connection.isValid((int) MILLISECONDS.toSeconds(Math.max(1000, validationTimeout)));
```

Comment:
If ``validationTimeout`` is an ``int``, we incur an additional unseen cast:
```java
MILLISECONDS.toSeconds(Math.max(1000, validationTimeout))
                       ^
                      here
```
Math.max() will return an ``int``.